### ISSUE TITLE
Running tests twice fails with `0x0x` in bytecode

### ISSUE THREAD

**Issue by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll** [counterfactual/monorepo#1461]
Running the following from a fresh clone errors out

```
yarn && yarn build && yarn test && cd packages/node && yarn test
```

with the following

```
invalid hexidecimal string (arg="value", value="0x0x608060405234801561001057600080fd5b506040516020806101a88339810180604052602081101561003057600080fd5b8101908080519060200190929190505050600073ffffffffffffffffffffffffffffffffffffffff168173ffffffffffffffffffffffffffffffffffffffff1614156100c7576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260248152602001806101846024913960400191505060405180910390fd5b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555050606e806101166000396000f3fe608060405273ffffffffffffffffffffffffffffffffffffffff600054163660008037600080366000845af43d6000803e6000811415603d573d6000fd5b3d6000f3fea165627a7a72305820694407262d1bf8a12b96a9ab0006c182bd726d829087da293c23bcb1e4bd03930029496e76616c6964206d617374657220636f707920616464726573732070726f7669646564", version=4.0.27)
```

**Comment by cf19drofxots** [comment]
Is that the only line that's displayed?

What test?

**Comment by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll** [comment]
```
yarn run v1.15.2
$ yarn run test:node && yarn run test:machine
$ jest --testPathIgnorePatterns=test/machine --setupFiles dotenv-extended/config --runInBand --bail --forceExit
 FAIL  test/integration/uninstall-virtual.spec.ts
  Node method follows spec - uninstall virtual
    Node A and C install a Virtual AppInstance through an intermediary Node B,then Node A uninstalls the installed AppInstance
      ✕ sends uninstall  (83ms)

  ● Node method follows spec - uninstall virtual › Node A and C install a Virtual AppInstance through an intermediary Node B,then Node A uninstalls the installed AppInstance › sends uninstall

    invalid hexidecimal string (arg="value", value="0x0x608060405234801561001057600080fd5b506040516020806101a88339810180604052602081101561003057600080fd5b8101908080519060200190929190505050600073ffffffffffffffffffffffffffffffffffffffff168173ffffffffffffffffffffffffffffffffffffffff1614156100c7576040517f08c379a00000000000000000000000000000000000000000000000000000000081526004018080602001828103825260248152602001806101846024913960400191505060405180910390fd5b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555050606e806101166000396000f3fe608060405273ffffffffffffffffffffffffffffffffffffffff600054163660008037600080366000845af43d6000803e6000811415603d573d6000fd5b3d6000f3fea165627a7a72305820694407262d1bf8a12b96a9ab0006c182bd726d829087da293c23bcb1e4bd03930029496e76616c6964206d617374657220636f707920616464726573732070726f7669646564", version=4.0.27)

      40 |     requestHandler: RequestHandler,
      41 |     params: Node.CreateChannelParams
    > 42 |   ): Promise<Queue[]> {
         |                                       ^
      43 |     return [requestHandler.getShardedQueue(CreateChannelController.methodName)];
      44 |   }
      45 |

      at Object.throwError (../../node_modules/ethers/errors.js:76:17)
      at Object.arrayify (../../node_modules/ethers/utils/bytes.js:53:20)
      at _pack (../../node_modules/ethers/utils/solidity.js:22:28)
      at ../../node_modules/ethers/utils/solidity.js:79:20
          at Array.forEach (<anonymous>)
      at Object.pack [as solidityPack] (../../node_modules/ethers/utils/solidity.js:78:11)
      at CreateChannelController.executeMethodImplementation (src/methods/state-channel/create/controller.ts:42:39)
      at executeCached (src/methods/controller.ts:9:32)
      at run (../../node_modules/p-queue/index.js:211:20)
      at PQueue._tryToStartAnother (../../node_modules/p-queue/index.js:173:25)
      at Promise (../../node_modules/p-queue/index.js:220:9)
      at PQueue.add (../../node_modules/p-queue/index.js:205:10)
      at CreateChannelController.executeMethod (src/methods/controller.ts:17:23)
```

**Comment by snario** [comment]
Similar to #669

**Comment by IIIIllllIIIIllllIIIIllllIIIIllllIIIIll** [comment]
fixed with #1524
